src/database/datatools/nomass_materials.py

```
@@ -0,0 +1,146 @@
+        cursor.execute("SELECT id FROM all_materials WHERE no_mass_material_id = ?", (material_id,))
+        am_row = cursor.fetchone()
+        if am_row is None:
+            conn.close()
+            raise ValueError(f"No all_materials entry found for no_mass_material_id {material_id}")
+        am_id = am_row['id']
+        cursor.execute("UPDATE all_materials SET name = ? WHERE id = ?", (name, am_id))
+        cursor.execute("UPDATE all_materials SET datetime = ? WHERE id = ?", (timestamp_int, am_id))
+    
+    conn.commit()
+    conn.close()
+    des_data = [material_id] + dt[:-1] 
```

Ruff (0.14.14)

[warning] Consider `[material_id, *dt[:-1]]` instead of concatenation

Replace with `[material_id, *dt[:-1]]`

(RUF005)